ledger/src/blockstore.rs

```diff
@@ -7166,51 +7141,50 @@ pub mod tests {
             .unwrap();
 
         let slot_meta = blockstore.meta(0).unwrap().unwrap();
-        let shred5 = shreds[5].clone();
-
```

Comment:
here we have tried to insert a single data shred making its own FEC set. With merkle shreds this does not work, there is a minimal amount of shred you need (32) to make a set with last_in_slot flag set, so we need to make that set rather than mutate a single shred

Reply:
> there is a minimal amount of shred you need (32) to make a set with last_in_slot flag set

Is this true / a byproduct of the fixed FEC set size change ? Ie, is `terminator.len() == 32` in the below code ? The point of `LAST_SHRED_IN_SLOT` signal shred was to have a cheap way to signal "I'm not going to continue this block, you can stop replay of it"

Reply:
no this has been true ever since we had merkle shreds. there was a special case for last batch for security related to turbine censoring. With 32:32 this is a moot point, its always 32:32 now. So if we just want to terminate block early, we need to make a full 64 shred set. However, this never happens in practice due to smart coalesce policy that Brennan put in place.